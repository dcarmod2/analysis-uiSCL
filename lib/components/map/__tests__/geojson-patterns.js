//
import enzyme from 'enzyme'
import React from 'react'

import Patterns from '../geojson-patterns'

describe('Components > Map > GeoJSON-Patterns', () => {
  it('renders correctly', () => {
    const props = {
      patterns: [],
      color: 'blue'
    }

    const tree = enzyme.shallow(<Patterns {...props} />)
    expect(tree).toMatchSnapshot()
  })
})
