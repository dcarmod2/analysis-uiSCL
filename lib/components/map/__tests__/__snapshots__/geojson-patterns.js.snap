// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Map > GeoJSON-Patterns renders correctly 1`] = `
<KeyedGeoJSON
  color="blue"
  data={
    {
  "type": "FeatureCollection",
  "features": []
}
  }
  weight={3}
/>
`;
